src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task to be added to Duke with a deadline.
+ */
+public class Deadlines extends Task {
+
+    protected String description;
+    private LocalDateTime dateTime;
+
+    public Deadlines(String description, LocalDateTime dateTime) {
+        super(description);
+        this.dateTime = dateTime;
+    }
+
+    /**
+     * Returns string of formatted deadline.
+     *
+     * @return Formatted string.
+     */
+    private String dateStr() {
+        String formatDateTime = this.dateTime.format(DateTimeFormatter.ofPattern("dd MMM yyyy HH:mm"));
```

Comment:
Should this be named `formattedDateTime`? I think the current name sounds like the name of a method.